python-avd/pyavd/_eos_designs/schema/schema_fragments/management_eapi.schema.yml

```
@@ -25,3 +25,44 @@ keys:
         default: true
       default_services:
         type: bool
+      vrfs:
+        # Todo: For 6.0 default should use use_default_mgmt_method_vrf and the backward compatibility behavior removed.
+        default:
+          - name: use_mgmt_interface_vrf
+            enabled: true
+        type: list
+        primary_key: name
+        description: |-
+          Note: For backward compatibility, the default behavior does not enforce `mgmt_ip` to be present in VRFs.
+          If this enforcement is desired, users must explicitly define VRFs. This behavior applies up to AVD version 6.0.
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: |-
+                VRF name.
+                The value will be interpreted according to these rules:
+                - `use_mgmt_interface_vrf` will configure the eAPI ACL under the VRF set with `mgmt_interface_vrf`.
```

Maybe we should be more generic here (as we may configure ACL here, but as well may not)

```suggestion
                - `use_mgmt_interface_vrf` will configure the eAPI under the VRF set with `mgmt_interface_vrf`.
```